FGO_REGULAR.lua

```diff
@@ -1,105 +1,103 @@
---Internal settings - do not modify.

```

Comment:
Please convert all Linux style endings (LF) to Windows style (CR LF)

Reply:
I double-checked and it seems that it is already in CRLF.
If you're wondering why git diff isn't showing properly, it should be because the file name was changed.

Reply:
Actually, it is LF on your repository. I downloaded the file from your repository and it didn't have CRs.

I think you have `core.autocrlf` activated for your repository, which converts all CRLFs into LFs when you commit the files and converts LFs into CRLFs when you load files from the repository.

Disable this by executing `git config core.autocrlf false` in your repository folder.

Reply:
Yup! it's set to true, so that's the problem. I'm sorry.
I'll commit changes in a few minutes

Reply:
@reconman asking every contributor to change default git settings is meh. Instead you can set up expected line ending in `.gitattributes` here in repo. See https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings#per-repository-settings

Reply:
I've changed line endings to CR LF and rebased into a single commit.

@z0rc That would be pretty nice.

Reply:
@z0rc You're right, I will set it up. I actually read that article shortly before you linked it.

@MathewSachin Sorry, I messed up in my own PR, where autocrlf was also enabled. Since there's no option to convert all line endings to CRLF on committing, I will instead force everyone to commit LFs.

Reply:
No problems, it happens sometimes, fine then I'll merge your changes from master to here and make sure to change back to LF

Reply:
@MathewSachin All files in the repository now use LF and are converted to CRLF if you're on Windows.

Now your branch has a merge conflict with master, because you want to commit CRLF endings. I'm very sorry for wasting your time.

Reply:
@MathewSachin Btw, you can convert all line endings automatically by running `git add --renormalize .`

Reply:
Well, I already did that manually 😆 
Now, there are many commits and I'm unable to rebase correctly.
If anyone merges this PR, it would be best to use the Rebase/Squash and merge option.